circle.yml

```diff
@@ -4,7 +4,25 @@ jobs:
   core:
     steps:
       - checkout
-      - run: ./gradlew testcontainers:check
+      - run:
+          command: ./gradlew testcontainers:check
+          environment:
+            TZ: "/usr/share/zoneinfo/ETC/UTC"
```

Comment:
I don't think this TZ env is needed anywhere but jdbc tests. Since some users check our CircleCI config for the inspiration, maybe we should remove it to keep it as small as possible?

Reply:
Yeah, I was really in two minds about it. I put it in all for consistency, but instead I'll trim it down and have a comment explaining why it's there,